packages/@aws-cdk/aws-cloudfront-origins/README.md

```diff
@@ -76,3 +76,20 @@ new cloudfront.Distribution(this, 'myDist', {
 ```
 
 See the documentation of `@aws-cdk/aws-cloudfront` for more information.
+
+## Failover Origins
```

Comment:
I feel this heading should at least parenthetically reference "Origin Groups", as that's what the feature is (inconsistently) called in CloudFront.
```suggestion
## Failover Origins (Origin Groups)
```

Reply:
Done!